lua/nvim_lsp/metals.lua

```diff
@@ -32,7 +38,7 @@ local function make_installer()
       os.execute("mkdir " .. install_dir)
       coursier_exe = "cs"
     end
-    local install_cmd = string.format("%s bootstrap --java-opt -Xss4m --java-opt -Xms100m --java-opt -Dmetals.client=coc.nvim org.scalameta:metals_2.12:latest.release -r bintray:scalacenter/releases -r sonatype:snapshots -o %s -f", coursier_exe, metals_bin)
```

Comment:
The `-Dmetals.client=coc.nvim` can now be removed since we are able to specify everything we need via `initializationOptions`.